mix.exs

```diff
@@ -57,6 +57,7 @@ defmodule SubwaysideUi.MixProject do
       {:hackney, "~> 1.18"},
       {:ex_aws_kinesis, "~> 2.0"},
       {:gen_stage, "~> 1.2"},
+      {:httpoison, "~> 2.0"},
```

Comment:
> question: what do you think about using Req instead (https://github.com/wojtekmach/req)? I'm personally trying to avoid new dependencies on HTTPoison/Hackney (though I haven't made this a policy so don't worry), both because I think Mint is a better foundation library, and because Mint/Finch/Req are pure Elixir (Hackney is Erlang).

Sounds good to me - I just used HTTPoison because it seemed to be the most in-use library in our code.